staging/provisioning/windows/windowssecuretls.ps1

```diff
@@ -0,0 +1,387 @@
+﻿#***************************************************************************************************************
+# Below functions are modified from TLSSettings.ps1

```

Comment:
Is the TLSSettings.ps1 mentioned here referring to the script on https://docs.microsoft.com/en-us/azure/cloud-services/applications-dont-support-tls-1-2#step-1-create-the-powershell-script-to-enable-tls-10-and-tls-11?
Should we add a reference here?

Reply:
It is very similar but it references an internal signing script.

Reply:
Would it be better to not reference the script here if it isn't public?

Reply:
The script is validated and should use the same settings as the link you provided.
I can add the internal link but others outside of Microsoft cannot open the link and it is very complex and may be changed later. 
@marosset how do you think?

Reply:
Remove the comment